vocabularies/UI.json

```diff
@@ -1228,7 +1228,7 @@
       "$Type": "UI.TextArrangementType",
       "$AppliesTo": ["Annotation", "EntityType"],
       "@Core.Description": "Describes the arrangement of a code or ID value and its text",
-      "@Core.LongDescription": "If used for a single property the Common.Text annotation is annotated"
+      "@Core.LongDescription": "This term can annotate a [`Common.Text`](Common.md#Text) annotation of the code or ID property whose value is the text."
```

Comment:
Can it also annotate `Common.ValueList`?

Reply:
It was never intended or documented to annotate Common.ValueList with UI.TextArrangement.